more_itertools/more.py

```diff
@@ -1897,7 +1895,7 @@ def getter(obj):
                 return obj[i]
             except IndexError:
                 # basically if we have an iterable like
-                # iter([(1, 2, 3), (4, 5), (6,)])
+                # iter([(1, 2, 3), (4, 5), (6,)])  # noqa: ERA001
```

Comment:
I object to `ERA001` being enabled. A Google search shows a high false positive rate.  Also, the premise of the rule, "commented-out code is bad" is itself dubious.

We really don't want to garbage up the code with a bug on `noqa`. That defeats the purpose of making the code more readable.